pytorch_lightning/accelerators/tpu.py

```diff
@@ -59,3 +59,18 @@ def _move_optimizer_state(self, device: Optional[torch.device] = None) -> None:
         for opt in self.optimizers:
             for p, v in opt.state.items():
                 opt.state[p] = apply_to_collection(v, torch.Tensor, move_data_to_device, self.root_device)
+
+    def get_device_stats(self, device: Optional[torch.device] = None) -> Dict[str, Any]:
+        """Gets stats for the given TPU device."""
+        device_stats = {}
+        memory_info = xm.get_memory_info(device)
```

Comment:
memory_info accepts a string: https://pytorch.org/xla/release/1.9/index.html#torch_xla.core.xla_model.get_memory_info

which is inconsistent with the typehint or `device` above